src/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class DotnetFiles
+    {
+        /// <summary>
+        /// The CLI ships with a .version file that stores the commit information and CLI version
+        /// </summary>
+        public static string VersionFile => Path.GetFullPath(Path.Combine(typeof(DotnetFiles).GetTypeInfo().Assembly.Location, "..", "..", ".version"));
```

Comment:
This won't work on Linux cause the .version file is missing at the moment: https://github.com/dotnet/cli/issues/1498